app/reducers/auth.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+import * as sc from 'spatialconnect/native';
+
+const initialState = {
+  token: null,
+  userName: null,
```

Comment:
Where would this get set?  I don't see it in the reducers so maybe we should remove it for now?